### PR TITLE
Switch to Apollo client

### LINKED ISSUE #219: Consider switching to Apollo client

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #219

### CODE REVIEW

File: portal/src/graphql/adminapi/UsersList.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      totalCount
+    }
+  }
+`;
+
+// There is createRefetchContainer and createPaginationContainer.
```
Comment (kiootic) [inline portal/src/graphql/adminapi/UsersList.tsx:114]: I think we can remove the top part of comment & rename the variable?

File: portal/src/AppRoot.tsx [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
 const AppRoot: React.FC = function AppRoot() {
   const { appID } = useParams();
-  const environment = useMemo(() => {
-    return makeEnvironment(appID);
+  const client = useMemo(() => {
+    return makeClient(appID);
   }, [appID]);
   return (
-    <AppContext.Provider value={environment}>
+    <ApolloProvider client={client}>
```
Comment (kiootic) [inline portal/src/AppRoot.tsx:14]: It seems this `ApolloProvider` (for admin API) is within another `ApolloProvider` (for portal backend API), can we call portal backend API using `useQuery` hook in the children components?

Comment (louischan-oursky) [reply portal/src/AppRoot.tsx:14]: Of course!

```typescript
import { client } from "../portal/apollo";

function MyComponent() {
  // Use client from current context
  const obj1 = useQuery(someAdminAPIQuery);
  // Use the provided client
  const obj2 = useQuery(somePortalQuery, { client });
}
```

**Comment by louischan-oursky** [comment]
Updated.

**Review by kiootic — APPROVED** [review]

**Pull request merged**